feature/topic/src/main/java/com/google/samples/apps/nowinandroid/feature/topic/navigation/TopicNavigation.kt

```diff
@@ -37,7 +37,9 @@ internal class TopicArgs(val topicId: String) {
 
 fun NavController.navigateToTopic(topicId: String) {
     val encodedId = Uri.encode(topicId)
-    this.navigate("topic_route/$encodedId")
+    this.navigate("topic_route/$encodedId") {
+        launchSingleTop = true
```

Comment:
👌

Reply:
@mmoczkowski Doesn't using `launchSingleTop` here introduce a bug where you can no longer do nested navigation from interests graph. 

For example, if I tap on a topic within a news card, it does not take me to that topic screen (when the code explicitly calls navigate to topic).